packages/bpk-component-switch/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "bpk-component-switch",
+  "version": "0.0.51",
```

Comment:
Doesn't matter as it will be released 1.0.0, but this is a very random value

Reply:
Yeah haha, so this is because Lerna bumps the version of the boilerplate component every time there's a release. It's never been worth fixing just because it's not actually causing a problem.